tests/manage/mcg/test_namespace.py

```diff
@@ -68,6 +69,26 @@ def test_namespace_bucket_creation(
             read_ns_resources=[ns_resource_name],
         )
 
+    @pytest.mark.polarion_id("OCS-2407")
+    @tier1
+    def test_namespace_bucket_creation_with_rgw(
```

Comment:
Skip test if not vsphere platform.

Reply:
It's skipped thanks to `rgw_deployments` fixture.

Reply:
ack